FreeRTOS/FreeRTOS
# UT: Add streambuffer test to make PR #391 pass

Add streambuffer test to make kernel PR #391 pass

Description
-----------
kernel PR 391 fails as a new assert is added without UT
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/391

Test Steps
-----------
add an test to check for the assert


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
